src/frontend/staff-portal/src/app/services/jj-dispute.service.ts

```diff
@@ -181,6 +181,19 @@ export class JJDisputeService {
       );
   }
 
+  // oracle To Date Format converts 2023-04-05T16:00:00+00:00 to MM/DD/YYYY HH:MM where using military 24hr
+  // OR 2023-04-05 to 04/05/2023
+  public toDateFormat(oracleDate: string): string {
```

Comment:
We are using custom date pipe, you can modify it if needed

Reply:
nope -- dont want to convert this string to date since it messes up the time zone